browser/resources/settings/brave_content_page/containers_icon.ts

```diff
@@ -0,0 +1,72 @@
+// Copyright (c) 2025 The Brave Authors. All rights reserved.
+// This Source Code Form is subject to the terms of the Mozilla Public
+// License, v. 2.0. If a copy of the MPL was not distributed with this file,
+// You can obtain one at https://mozilla.org/MPL/2.0/.
+
+import { CrLitElement } from 'chrome://resources/lit/v3_0/lit.rollup.js'
+
+import { Icon } from '../containers.mojom-webui.js'
+import { getCss } from './containers_icon.css.js'
+import { I18nMixinLit } from '//resources/cr_elements/i18n_mixin_lit.js'
+import { getHtml } from './containers_icon.html.js'
+
+export type IconSelectedEvent = CustomEvent<{
+  icon: Icon
+}>
+
+const SettingsBraveContentContainersIconElementBase = I18nMixinLit(CrLitElement)
```

Comment:
nit: I don't think this needs the `I18nMixinLit`, as we don't use `i18n` anywhere :smile: